gnomad_constraint/utils/constraint.py

```diff
@@ -467,6 +470,9 @@ def apply_models(
         .aggregate(**agg_expr)
     )
 
+    # TODO: Remove repartition once partition_hint bugs are resolved
+    ht = ht.repartition(expected_variant_partition_hint)
```

Comment:
Just want to confirm that the ht doesn't need a repartition to `obs_pos_count_partition_hint` after the call to `create_observed_and_possible_ht`

Reply:
i think it will run fine with these edits but can make more changes if it fails